scripts/iam_policy.json

```diff
@@ -23,14 +23,24 @@
           "kms:*",
           "states:*",
           "backup:*",
-          "backup-storage:MountCapsule"
+          "backup-storage:MountCapsule",
+          "ssm:GetParameter"
```

Comment:
Not needed.

This file is for permissions needed to deploy the solution. Not for permissions that are needed at runtime

Reply:
When customers want to run the code locally for development purposes, they would need to run `sls offline start` with an AWS account. The "offline" lambda that is spun up would use that AWS account to run FWoA code. That AWS account needs permission to grab parameters from SSM and to also invoke the API GW of  IntTran. 

I'll update the doc here to be more specific as to which AWS account is required when spinning up FWoA in `offline` mode.

https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/DEVELOPMENT.md